src/Data/Vec/Properties.agda

```diff
@@ -52,6 +53,19 @@ private
     m n o : ℕ
     ws xs ys zs : Vec A n
 
+------------------------------------------------------------------------
+-- Properties to List
+
+toList-injective
+  : ∀ {m n}
+  → .(m=n : m ≡ n)
```

Comment:
Can we follow the style guide about the placement of arrows on new lines?

Also can the name of this proof use `≡` instead of `=`?